caddyhttp/httpserver/replacer.go

```diff
@@ -249,7 +251,9 @@ func (r *replacer) getSubstitution(key string) string {
 	case "{uri_escaped}":
 		return url.QueryEscape(r.request.URL.RequestURI())
 	case "{when}":
-		return time.Now().Format(timeFormat)
+		return now().Format(timeFormat)
+	case "{whenISO}":
```

Comment:
How do you feel about `{when_iso}` to be consistent with a couple of other placeholders?